### DISCUSSION TITLE
enable x11 server via XQuartz

### DISCUSSION THREAD

**Discussion by tomkoi** [XQuartz/XQuartz#362] [Q&A] [answered]
Hello, I recently moved from Windows to Mac.
on Windows, I use some x11 app that creates an x11 server on my machine.
now on Mac, i understand I need to use XQuartz to do the same.
but my main goal here it's to make it work with SecureCRT.
can someone please guide me on how to configure it properly?

**Comment by jeremyhu** [comment]
I'm not sure what you mean by having it work with SecureCRT.  Can you elaborate?

**Reply by tomkoi** [reply, marked as answer]
> I'm not sure what you mean by having it work with SecureCRT. Can you elaborate?

Hi @jeremyhu, it's a 3-year-old bug; I already managed to work with this without a problem, closing the ticket.
